lib/sycamore/sycamore/tests/unit/writers/test_base.py

```diff
@@ -2,7 +2,7 @@
 from pathlib import Path
 from sycamore.data.document import Document, MetadataDocument
 from sycamore.plan_nodes import Node
-from sycamore.writers.base import BaseDBWriter
+from sycamore.connectors.writers.base import BaseDBWriter
```

Comment:
ok I'm on board with the 'everything to connectors'. I'm gonna make you move this file to test/unit/connectors/writers/whatever to reflect that tho

Reply:
Fair enough, done.